src/modules/modal/populateComments.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
```

Comment:
I would recommend you don't disable linters, those are important to write cleaner code.